.gitmodules

```diff
@@ -80,7 +80,7 @@
 	url = https://github.com/sonic-net/sonic-mgmt-framework
 [submodule "src/sonic-telemetry"]
```

Comment:
The submodule name is still sonic-telemetry. Is it easy to change this one in the same PR?